pkg/proxy/util/endpoints.go

```diff
@@ -53,12 +53,12 @@ func PortPart(s string) (int, error) {
 	// Must be IP:port
 	_, port, err := net.SplitHostPort(s)
 	if err != nil {
-		klog.Errorf("Error parsing '%s': %v", s, err)
+		klog.ErrorS(err, "failed to parse host-port", "input", s)
 		return -1, err
 	}
 	portNumber, err := strconv.Atoi(port)
 	if err != nil {
-		klog.Errorf("Error parsing '%s': %v", port, err)
+		klog.ErrorS(err, "failed to parse port", "input", port)
```

Comment:
Shouldn't we use "port" instead of "input"?
cc: @thockin @serathius

Reply:
/hold

Wait for the commit content to be resolved.